hazelcast/src/test/java/com/hazelcast/config/ConfigTest.java

```diff
@@ -328,6 +328,10 @@ public String getName() {
     }
 
     private static String getSimpleXmlConfigStr(String ...tagAndVal) {
+        if (tagAndVal.length % 2 != 0) {
+            throw new IllegalArgumentException("The number of tags and values parameters is odd." +
+                    " Please provide these tags and values as pairs.");
+        }
```

Comment:
Won't it be an NPE if we don't pass any parameter here? I guess we need a null check beforehand.

Reply:
Thanks for mentioning this. When I searched, I found this: https://stackoverflow.com/questions/17702807/java-3-dots-parameter-varargs-behavior-when-passed-no-arguments-or-null. It seems that when no parameter passed into it, it seems to create empty array (the `testLoadEmptyYamlFromStream` test above also passes on my machine), but it can be also null in some cases. I hadn't thought about it before you mentioned, because that check already hadn't been done before (we were checking for zero length array). I'll add this check.